earth_enterprise/src/third_party/apache2/apache-gee-httpd-ssl.conf.in.patch

```diff
@@ -0,0 +1,108 @@
+--- httpd-2.2.32.orig/docs/conf/extra/httpd-ssl.conf.in	2015-05-27 11:59:59.000000000 -0700
```

Comment:
Looks like this file is still included after the rename. It should be removed from the commit.